recipes-graphics/userland/userland/0017-khronos-backport-typedef-for-EGL_EXT_image_dma_buf_i.patch

```diff
@@ -0,0 +1,31 @@
+From ffc50f94bd172d02042d334464eb911c1c4ff47f Mon Sep 17 00:00:00 2001
+From: Hugo Hromic <hhromic@gmail.com>
+Date: Sun, 13 May 2018 10:49:04 +0100
+Subject: [PATCH] khronos: backport typedef for EGL_EXT_image_dma_buf_import
+
+The `gstreamer1.0-plugins-base` package version `1.14` uses `EGL_EXT_image_dma_buf_import`.
+This import expects the following typedef to be present in Khronos,
+however userland provides an older version without it.
+
+    typedef khronos_uint64_t EGLuint64KHR;
+
+This patch backports the missing typedef into the Khronos version in userland.
+
+Upstream-Status: Pending
```

Comment:
I don't see a corresponding upstream PR at https://github.com/raspberrypi/userland/pulls ... can you please provide a link (to a PR or mailing list archive post) back to the pending submission? Or did you mean Khronos as the upstream here?

Reply:
Yes I meant Khronos as the upstream, in particular here: https://www.khronos.org/registry/EGL/api/EGL/eglext.h
I haven't yet submit a patch to userland.
Should I clarify this further in the patch?

Reply:
According to the guidelines (https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines), the `Pending` status also covers not yet submitted patches, hence why I used it:
```
Pending
  - No determination has been made yet or not yet submitted to upstream
```